eclair-cli/src/main/java/fr/acinq/eclair/cli/payment/SendToRoute.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 ACINQ SAS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.acinq.eclair.cli.payment;
+
+import fr.acinq.eclair.cli.utils.BaseSubCommand;
+import okhttp3.ResponseBody;
+import picocli.CommandLine;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static fr.acinq.eclair.cli.utils.Utils.print;
+
+@CommandLine.Command(name = "sendtoroute", description = "Sends money to a node without an invoice. The payment will be retried several times if needed.", sortOptions = false)
```

Comment:
nit: wrong copy-paste: should be "sends money to a user-defined route without an invoice"